### PR TITLE
Update mssql-setup.md

### PR DESCRIPTION
**Pull request by mikaelene**
I have changed the author back to me. Someone changed it a year or so ago. But I am the only creator and the sole owner of the PyPi-package. I kind of donated the adapter to the dbt-msft community.

## Checklist
- [x] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [x] I have added checklist item(s) to this list for anything anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

File: website/docs/docs/core/connect-data-platform/mssql-setup.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ description: "Read this guide to learn about the Microsoft SQL Server warehouse
 id: "mssql-setup"
 meta:
   maintained_by: Community
-  authors: 'dbt-msft community (https://github.com/dbt-msft)'
+  authors: 'Mikael Ene'
```
Comment (dataders) [inline website/docs/docs/core/connect-data-platform/mssql-setup.md]:
```suggestion
  authors: 'Mikael Ene & [dbt-msft community](https://github.com/dbt-msft)'
```

**Review by dataders — COMMENTED** [review]
hey @mikaelene! long time no talk. happy to have you credited. Does crediting both make sense?

My concern with giving you sole credit would be that folks might think that you're still an active maintainer of the project and bug you for updates.

How's my suggestion land with you?

File: website/docs/docs/core/connect-data-platform/mssql-setup.md (hunk 1 above)
Comment (mikaelene) [reply website/docs/docs/core/connect-data-platform/mssql-setup.md]: Hi @dataders . Long time yes :-). I think _maintained_by:_ explains who are maintaining the project. If the dbt-msft community are maintaining this, it should be stated there I guess? I mean, I did created this adapter and was the only maintainer for the first year. Previously my name was stated as author but was just removed one day, without any notification. It is a little weird to change author according to who currently maintains the project don't you think?

**Comment by mirnawong1** [comment]
Hey @dataders just checking in to see if the proposal works?

**Review by dataders — APPROVED** [review]
let's ship this as is.

**Review by mirnawong1 — APPROVED** [review]

File: website/docs/docs/core/connect-data-platform/mssql-setup.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ description: "Read this guide to learn about the Microsoft SQL Server warehouse
 id: "mssql-setup"
 meta:
   maintained_by: Community
-  authors: 'dbt-msft community (https://github.com/dbt-msft)'
+  authors: 'Mikael Ene & dbt-msft community(https://github.com/dbt-msft)'
```
Comment (mirnawong1) [inline website/docs/docs/core/connect-data-platform/mssql-setup.md]:
```suggestion
  authors: 'Mikael Ene & dbt-msft community (https://github.com/dbt-msft)'
```

**Pull request merged**